pyfar/plot/shortcuts/shortcuts.json

```diff
@@ -56,28 +63,33 @@
     "zoom_y_out": {
         "key": ["shift+down"],
         "info": "zoom in y-axis"},
-    "zoom_cm_in": {"key": ["*"],
+    "zoom_cm_in": {
+        "key": ["*", "alt+shift+up"],
         "info": "zoom colormap range in"},
     "zoom_cm_out": {
-        "key": ["_"],
+        "key": ["_", "alt+shift+down"],
```

Comment:
The hotkeys `*, _` are not very intuitive for me. Is there a reason for them? I just could not think of a mnemonic, etc. Might be I'm simply missing something. :)

Reply:
My general idea (for qwertz) was to have the arrow keys to control the axis and +/- for the colormap. Using the keys without a modifier moves things, using shift+key zooms things. I thought this might be easy to memorize.

`_` and `*` are `shift+minus` and `shift+plus` on qwertz. I added a `"key_verbose"` field for this case to make it more clear. The verbose key field is used for displaying the shortcuts if it exists.